release.py

```diff
@@ -0,0 +1,169 @@
+#! /usr/bin/env python3
+#
+# Copyright (c) 2020 Linaro Limited
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import argparse
+import netrc
+import os
+import requests
+import re
+
+from datetime import datetime
+
+
+JIRA_HOST = 'zephyrprojectsec.atlassian.net'
+BASEURL = f"https://{JIRA_HOST}/rest/api/2/"
+
+# Zephyr JIRA custom field names.
+CVE_FIELD = 'customfield_10035'
+EMBARGO_FIELD = 'customfield_10051'
+
+ISSUES = {}
+
+# Get authentication information.
+
+
+def get_auth(host):
```

Comment:
This is ok for now, but we should probably make a little module here for this shared code, instead of having a copy in each file.